content/docs/configuration.md

```diff
@@ -24,7 +24,7 @@ Both Packit-as-a-Service and packit tool use this configuration file.
  Key name                  | Type            | Description
 ---------------------------|-----------------|----------------------------------------------------------------------
  `specfile_path`           | string          | relative path to a spec file within the upstream repository (mandatory)
- `upstream_project_name`   | string          | name of the upstream repository (e.g. in PyPI); this is used in `%prep` section to generate an archive
```

Comment:
can you please keep this one here and say it's deprecated since packit 0.7.0?

Reply:
done